qa/integration-tests/src/test/java/io/camunda/it/utils/CamundaMultiDBExtension.java

```diff
@@ -285,20 +285,45 @@ private void injectFields(
   @Override
   public void afterAll(final ExtensionContext context) {
     if (databaseType == DatabaseType.ES || databaseType == DatabaseType.OS) {
-      final URI deleteEndpoint = URI.create(String.format("%s/%s-*", DEFAULT_ES_URL, testPrefix));
-      final HttpRequest httpRequest = HttpRequest.newBuilder().DELETE().uri(deleteEndpoint).build();
       try (final HttpClient httpClient = HttpClient.newHttpClient()) {
-        final HttpResponse<String> response = httpClient.send(httpRequest, BodyHandlers.ofString());
-        final int statusCode = response.statusCode();
+        // delete indices
+        // https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-delete-index.html
+        final URI deleteIndicesEndpoint =
+            URI.create(String.format("%s/%s-*", DEFAULT_ES_URL, testPrefix));
+        HttpRequest httpRequest =
```

Comment:
:thought_balloon: I'm wondering if we add few retries or not to reduce flakyness. If a request fails for some reason, then the indices will not be deleted. Can this happen in case OS is overloaded?

Reply:
@entangled90 I will create a follow-up pr

Reply:
https://github.com/camunda/camunda/pull/28005